### PR TITLE
add secrets.rego and add in master-compliance-test

### PR DESCRIPTION
**Pull request by shahinshirvani**

### CODE REVIEW

File: aws/iac/master-compliance-test.json [hunk 1]
```diff
@@ -2575,6 +2575,58 @@
...
+                    "evals": [
+                        {
+                            "id":"PR-AWS-0028-RGX",
+                            "eval": "data.rule.gl_aws_secrets",
+                            "message": "data.rule.gl_aws_secrets_err",
+                            "remediationDescription":"",
+                            "remediationFunction":""
+                        }
+                    ],
+                    "title": "",
```
Comment (farchide) [inline aws/iac/master-compliance-test.json:2595]: title: `Secrets should be removed from the source code`

File: aws/iac/master-compliance-test.json [hunk 2]
```diff
@@ -2575,6 +2575,58 @@
...
+                    "description": "",
```
Comment (farchide) [inline aws/iac/master-compliance-test.json:2596]: Description: `We should not have any secret in the source code. Make sure to put the secrets in a vault`

**Review by farchide — CHANGES_REQUESTED** [review]
@shahinshirvani update the title and description with the items provided

**Comment by shahinshirvani** [comment]
i updated those info

**Review by farchide — APPROVED** [review]

**Pull request merged**